content/api/usage/usage_multi_org_code.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Get Usage across your Multi-Org Account
+type: apicode
+order: 22.4
+external_redirect: /api/#get-usage-across-your-multi-org-account
+---
+
+##### Signature
+`GET https://app.datadoghq.com/api/v1/usage/summary`
+##### Example Request
+{{< code-snippets basename="api-billing-usage-summary" >}}
```

Comment:
It seems that there is no code snippets file for this end-point. 
Could you add at least the sh example and add disclaimers for py and rb code snippets? many thanks 🙇

Reply:
@l0k0ms Because its in private beta we haven't included code snippets for any of the usage endpoints